pages/index.js

```diff
@@ -1,102 +1,155 @@
 
-const validatorMapper = {}
+import schema from "../schemas/schema";
+
+const validatorMapper = {};
 
 const useStyles = makeStyles((theme) => ({
   paper: {
     marginTop: theme.spacing(8),
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
   },
   avatar: {
     margin: theme.spacing(1),
     backgroundColor: theme.palette.secondary.main,
   },
   form: {
-    width: '100%', // Fix IE 11 issue.
+    width: "100%", // Fix IE 11 issue.
     marginTop: theme.spacing(1),
   },
   submit: {
     margin: theme.spacing(3, 0, 2),
-  }
-
+  },
 }));
 
 const theme = createMuiTheme({
   overrides: {
     MuiButton: {
       root: {
-        background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+        background: "linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)",
         borderRadius: 3,
         border: 0,
-        color: 'white',
+        color: "white",
         height: 48,
-        padding: '0 30px',
-        boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+        padding: "0 30px",
+        boxShadow: "0 3px 5px 2px rgba(255, 105, 135, .3)",
       },
     },
     MuiCheckbox: {
       root: {
-        padding: '1px 9px'
-      }
-    }
+        padding: "1px 9px",
+      },
+    },
   },
 });
 
-const MyFormTemplate = (props) => <FormTemplate {...props} showFormControls={false} />
+const MyFormTemplate = (props) => (
+  <FormTemplate {...props} showFormControls={false} />
+);
 
 const validate = (values) => {
-  console.log(values)
+  console.log(values);
   const errors = {};
 
-  for(let i=1; i<=17; i++){
-    if (values.SDGs && values.SDGs.includes(i) && ! values['evidenceText'+i] && ! values['evidenceURL'+i]) {
-      errors['evidenceText'+i] = 'Either the description or a URL is required';
-      errors['evidenceURL'+i] = 'Either the description or a URL is required';
+  for (let i = 1; i <= 17; i++) {
+    if (
+      values.SDGs &&
+      values.SDGs.includes(i) &&
+      !values["evidenceText" + i] &&
+      !values["evidenceURL" + i]
+    ) {
+      errors["evidenceText" + i] =
+        "Either the description or a URL is required";
+      errors["evidenceURL" + i] = "Either the description or a URL is required";
     }
   }
 
   return errors;
+};
+
+function openPR(values) {
+  const myJSON = JSON.stringify(values, null, "\t");
+
+  // Return project json file aligned with naming convention
+  let name = values.name.toLowerCase() + ".json";
+
+  const MyOctokit = Octokit.plugin(createPullRequest);
+
+  const TOKEN = process.env.ACCESS_TOKEN; // create token at https://github.com/settings/tokens/new?scopes=repo
+  const octokit = new MyOctokit({
+    auth: TOKEN,
+  });
+
+  // Returns a normal Octokit PR response
+  // See https://octokit.github.io/rest.js/#octokit-routes-pulls-create
+  octokit
+    .createPullRequest({
+      owner: "nathanbaleeta",
```

Comment:
Let's make these 2 variables at the top of the file

Reply:
Resolved in commit 225d4f0.